app/assets/javascripts/edit_work_utils.js

```diff
@@ -169,12 +169,12 @@ $(() => {
 
   function deletePerson(rowToDelete, type) {
     const rowExists = $(rowToDelete).length > 0;
-    const rowData = $(`${rowToDelete} input:not(.hidden)`);
+    const $rowData = $(`${rowToDelete} input:not(.hidden)`);
     let i; let
       token;
     let rowText = '';
-    for (i = 0; i < rowData.length; i++) {
-      token = $(rowData[i]).val();
+    for (i = 0; i < $rowData.length; i++) {
+      token = $($rowData[i]).val();
```

Comment:
Rewrapping with jQuery is not necessary, but didn't want to change too many things at once.